tests/unittest/cloud_governance/common/clouds/aws/athena/test_athena_operations.py

```diff
@@ -0,0 +1,15 @@
+from cloud_governance.common.clouds.aws.athena.abstract_athena_operations import AbstractAthenaOperations
+from cloud_governance.common.clouds.aws.athena.pyathena_operations import PyAthenaOperations
```

Comment:
What about test for boto3 client athena class ?

Reply:
No need, one method is enough.

Reply:
I created two patches, that wrapped by mock_athena

Reply:
its not clear because the mock class with different name from original one